helm/charts/goquorum-node/templates/node-storage.yaml

```diff
@@ -50,7 +50,7 @@ parameters:
 #   gidRangeEnd: "2000" # optional
 #   basePath: "/dynamic_provisioning" # optional
 
-{{- else }}
+{{- else if eq .Values.cluster.provider "longhorn" }}
```

Comment:
don't need to use storageClass in case of `cluster.provider: local` .
https://kubernetes.io/blog/2017/03/dynamic-provisioning-and-storage-classes-kubernetes/

Remove storageClass in node-statefulset.yaml